test/cases/compile_errors/slicing_single-item_pointer_with_len_1.zig

```diff
@@ -0,0 +1,11 @@
+export fn entry(ptr: *i32) void {
+    const slice = ptr[0..];
+    _ = slice;
+}
+
+// error
+// backend=stage2
+// target=native
+//
+// :2:22: error: slice of single-item pointer
+// :2:22: note: single-item pointer can be coerced to array using '@as(*[1]i32, ptr)'
```

Comment:
`*[1]i32` is not an array, it is a pointer to an array. Compile errors must be pedantically precise.